api/src/public/login/script.js

```diff
@@ -125,6 +125,10 @@ const getRedirectUrl = function() {
   }
 };
 
+const getSSOLoginUrl = function(){
```

Comment:
When does this get invoked? From local tests, the controller code seems to take precedence.

Reply:
This method is being invoked when the ssoLoginButton is clicked. Still on the same file, on line 221
```
  const ssoLoginButton = document.getElementById('login-sso');
  ssoLoginButton.addEventListener('click', function(){
    getSSOLoginUrl();
  });
```

Reply:
I see. Which in turn invokes the function in the controller via the /medic/login/oidc route.